src/Symfony/Component/Routing/CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ CHANGELOG
    Instead of overwriting them, use `__serialize` and `__unserialize` as extension points which are forward compatible
    with the new serialization methods in PHP 7.4.
  * exposed `utf8` Route option, defaults "locale" and "format" in configuration loaders and configurators
- * added support for invokable route loader services
+ * added support for invokable service route loaders
```

Comment:
In https://github.com/symfony/symfony/pull/30926, there were some discussions about the name. The name of a service that provide routes is a "service route loader". cf https://symfony.com/doc/current/routing/custom_route_loader.html + https://github.com/symfony/symfony/blob/fcb330905d0d9d1f92345761e5ace96a09214308/src/Symfony/Component/Routing/Loader/ObjectRouteLoader.php#L53. In https://github.com/symfony/symfony/pull/30933, I unfortunately made the mistake to introduce a new name "route loader services" that is now in the doc. I will fix everything to be "service route loader".